apps/qubit/modules/informationobject/templates/_identifierOptions.php

```diff
@@ -0,0 +1,11 @@
+<div class="section">
```

Comment:
One thing about this partial is that the alternative identifiers partial is not used in the DC and MODS templates, so the "Add alternative identifier(s)" link shouldn't appear in there, and placed in here it will (without working). If you need the two links together in the GUI you could send an option to this template with a boolean to show the alternative identifiers link or not based on the IO template.